generators/client/templates/test/e2e/sample.test.js

```diff
@@ -1,24 +1,10 @@
-var URL = "https://localhost:8000/<%= applicationFolder %>/";
+const URL = "https://localhost:8000/<%= applicationFolder %>/";
```

Comment:
If you delete the e2e tests, this file will no longer be used.

Command `e2e` in `package.json` must also be removed.